src/generator.cs

```diff
@@ -5639,6 +5641,15 @@ public void PrintAdviceAttribute (ICustomAttributeProvider mi)
 		print ($"[Advice ({Quote (p.Message)})]");
 	}
 
+	public void PrintRequiresSuperAttribute (ICustomAttributeProvider mi)
+	{
+		var p = AttributeManager.GetCustomAttribute<RequiresSuperAttribute> (mi);
```

Comment:
Nitpick: there's `AttributeManager.HasAttribute`, which is faster since it doesn't require creating attribute instances: https://github.com/xamarin/xamarin-macios/blob/07c24949c0b846cd1e65f77d13c68dc34f3ad496/src/generator-attribute-manager.cs#L258